### ISSUE TITLE
🐛 BUG: Tab navigation doesn't cause scrolling inside post content area

### ISSUE THREAD

**Issue by RafidMuhymin** [dilmaheu/dilmahtea.me#253]
**Describe the bug**
Tab navigation doesn't cause scrolling inside the content area of a post details page.

**Steps to Reproduce**

1. Go to https://dilmahtea.me/en/healthy-waffles/
2. Navigate using the `tab` key
3. See that after reaching the last interactive element before the content area, tab navigation doesn't cause scrolling.

**Expected behavior**
Scolling should work correctly the user should be able to see where he's being taken to.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 110

**Additional context**
N/A
